src/webots/gui/WbMainWindow.cpp

```diff
@@ -1598,18 +1599,30 @@ void WbMainWindow::exportVrml() {
   simulationState->resumeSimulation();
 }
 
-void WbMainWindow::exportHtml() {
+void WbMainWindow::exportHtmlFiles() {
   WbSimulationState::Mode currentMode = WbSimulationState::instance()->mode();
-  WbWorld *world = WbWorld::instance();
 
-  QString fileName = findHtmlFileName("Export HTML Scene");
+  QString fileName = findHtmlFileName("Export HTML file");
   if (fileName.isEmpty()) {
     WbSimulationState::instance()->setMode(currentMode);
     return;
   }
 
   if (WbProjectRelocationDialog::validateLocation(this, fileName)) {
-    world->exportAsHtml(fileName, false);
+    const QFileInfo info(fileName);
+    QStringList extensions = {".html", ".x3d"};
+    if (QFileInfo(WbStandardPaths::webotsTmpPath() + "export_cloud.json").exists())
```

Comment:
I would rename "export_cloud" to "cloud_export" everywhere:
```suggestion
    if (QFileInfo(WbStandardPaths::webotsTmpPath() + "cloud_export.json").exists())
```